libs/features/branch-quick-links/branch-quick-links.js

```diff
@@ -0,0 +1,81 @@
+import { getMetadata, loadStyle, getConfig, createTag } from '../../utils/utils.js';
+
+function addLoader(elem, text) {
+  const { base } = getConfig();
+  loadStyle(`${base}/features/branch-quick-links/branch-quick-links.css`);
+  const lbl = createTag('div', { class: 'pbar-label' });
+  lbl.textContent = text;
+  const tr = createTag('div', { class: 'pbar-value' });
+  tr.style.display = 'block';
+  const pb = createTag('div', { class: 'pbar' }, tr);
+  pb.style.width = '100%';
+  pb.style.height = '10px';
+  pb.style.borderRadius = '5px';
+  pb.style.backgroundColor = '#D5D5D5';
+  pb.style.marginTop = '10px';
+  pb.style.overflow = 'hidden';
+  const container = createTag('div', { class: 'pbar-container' }, [lbl, pb]);
```

Comment:
When you just read this line, `[lbl, pb]` is getting quite confusing for someone who reads this for the first time and didn't look at a lot more within the function.